transform.ml

```diff
@@ -119,8 +121,12 @@ let optimize (opts : string list) (prog : program) : program option =
       as_opt_program const_fold
     | "prune" ->
       as_opt_program branch_prune
-    | "inline" ->
-      Transform_inline.inline
+    | "inline_max" ->
```

Comment:
There are some typos in the commit message (or code):

`inlining_*` -> `inline_*`
`inlining_map` -> `inline_max`

Is `inline_med` to a depth of 3 or 4?